assets/views/home/header.ejs

```diff
@@ -33,6 +33,7 @@
             <li><a href="#" data-toggle="modal" data-target="#flairAppList">Flair App List</a></li>
             <li><a href="#" data-toggle="modal" data-target="#flairMod">Edit Flairs</a></li>
             <li><a href="/mod/banuser">Ban a User</a></li>
+            <li><a target="_self" href="/mod/downloadModmail" download="modmails.txt">Download Modmails</a></li>
```

Comment:
I thought it would be nice to have a link in case people want to download the entire archive. That way we can take the dropbox link offline, so we don't have to worry about people accidentally leaking it.

Keep in mind that this button doesn't the old, hard-to-parse txt file - it downloads a a file in JSON format directly from our database.

Also, I'm not sure why the other thing would be less steps -- this is a one-click download from the header.